package.json

```diff
@@ -82,5 +90,10 @@
     "@tailwindcss/language-service": {
       "postcss": "^8.0.0"
     }
-  }
+  },
+  "main": "index.js",
+  "directories": {
+    "example": "examples"
+  },
+  "types": "./index.d.ts"
 }
```

Comment:
What’s with all these metadata changes? None of the changes here are needed. I also noticed you published your fork as `monaco-tailwind` and didn’t update any of the metadata? Forking is fine, publishing too, but using such a similar name to publish a fork doesn’t seem like the nices thing to do. It’s also misleading to both your and my users to point your metadata at my project.

Reply:
Yeah, as I said, this is my very first time contributing, doing PR. I've published a package to test it on the project where I use Monaco. I only later discovered that I could have tested it directly with the demo in the repo. My bad on that. As I knew I would eventually delete the package, I didn't bother to update the metadata accordingly. So no worries about that, I'll update it and delete the package if the changes get merged here.